example/tile.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const LifxClient = require('../lib/lifx').Client;
+
+const client = new LifxClient();
+
+const LOOPTIME = 10000;
+
+function getBits(light, idx, chain) {
+  if (idx >= chain.total_count) {
+    console.log('All Bits get');
+    setTimeout(() => setBits(light, 0, chain), LOOPTIME);
+    return;
+  }
+  light.getTileState64(idx, 64, 0, 0, 8, (err) => {
+    if (err) {
+      console.error('getTileState64:', err);
+      return;
+    }
+    getBits(light, idx + 1, chain);
+  });
+}
+
+function setBits(light, idx, chain) {
```

Comment:
As this example has quite a lot going on, it would be great if you could add a comment describing what to expect when this starts running. Maybe include some `console.log`'s printing the values passed to `setTileState64` for example.

Reply:
done